docs/regression-test/regression-test.md

```diff
@@ -0,0 +1,44 @@
+# Regression Test
+
+## Introduction
+
+Regression test runs docfx e2e build locally against real docs repo to:
+- guard unexptected changes
+- ensure expected changes
+
+The test baseline is previous **commit build's output**, which is checked in [docfx.testdata](https://ceapex.visualstudio.com/Engineering/_git/docfx.testdata).  
+Each repo has its dedicated branch with the same repo name. e.g. *repo*: [https://github.com/MicrosoftDocs/azure-docs-pr](https://github.com/MicrosoftDocs/azure-docs-pr). *test-data*: [azure-docs-pr](https://ceapex.visualstudio.com/Engineering/_git/docfx.testdata?path=%2F&version=GBazure-docs-pr&_a=contents)
+
+For each PR build, the output will be compared with checked in baseline, and the diff will be commented along with the PR thread if any.
+
+## Selected Docs Repos
+Repos are selected for covering different docfx build functionalities. e.g.: the core markdown parser, yml SDP, versioning & xref, reference plugins, etc.
+
+| page view | repos                         | version | loc | xref | conceptual | SDP conceptual | SDP Ref | multi<br/>docset | Ecma2Yaml | Maml2Yaml | JoinTOC | Split<br/>TOC | Rest | vsts | learn | JS TS | special |
```

Comment:
How do you propose keeping this table in sync with the matrix inside `azure-pipeline-v3.yml`? The matrix seems like a better place as source of truth.

Reply:
The matrix lack the purpose of each repo. It's easy to edit the excel and update the markdown accordingly. Besides, this list won't be often updated.

Reply:
In general it is not a good practice to check in binary files to repos, especially now the same list has replicate 3 places: the matrix/markdown and excel.

Reply:
Agreed.  We can use *csv* format for easier editing.  

But do you have better solution for achieving following without data duplication?
- have a clear brief view of test repos we have and testing purposes of each repo (simply yml file can't provide this)
- the repo list can be easily updated